src/shared/ui/button/button.tsx

```diff
@@ -0,0 +1,21 @@
+import { clsx } from 'clsx'
+
+import type { ButtonHTMLAttributes } from 'react'
+
+interface Props extends ButtonHTMLAttributes<HTMLButtonElement> {
+  variant?: 'primary'
+}
+
+export default function Button({ children, variant = 'primary', className, ...props
+}: Props) {
+  const baseStyle = 'w-24 h-10'
+  const variantStyle = {
+    primary: 'bg-blue-500 text-white'
+  }
+
+  return (
+    <button className={
+      clsx(
```

Comment:
```suggestion
      cn(
```

cn에서 clsx를 사용하고 있어요! cn을 사용해주는게 좋아보입니다.